libraries/plugins/market_history/include/steemit/plugins/market_history/market_history_plugin.hpp

```diff
@@ -47,15 +47,17 @@ class market_history_plugin : public plugin< market_history_plugin >
 
       static const std::string& name() { static std::string name = STEEM_MARKET_HISTORY_PLUGIN_NAME; return name; }
 
+      flat_set< uint32_t > get_tracked_buckets() const;
+      uint32_t get_max_history_per_bucket() const;
+
       virtual void set_program_options(
          options_description& cli,
-         options_description& cfg );
-      void plugin_initialize( const variables_map& options );
-      void plugin_startup();
-      void plugin_shutdown();
+         options_description& cfg ) override;
 
-      flat_set< uint32_t > get_tracked_buckets() const;
-      uint32_t get_max_history_per_bucket() const;
+   protected:
```

Comment:
Is there a reason why these methods are protected? They aren't protected anywhere else.